main/githooks.py

```diff
@@ -306,7 +306,17 @@ def _test(input, output=None):
 
 
 def trim_trailing_whitespace_in_file(filename, new_file, in_place):
-    '''Remove trailing white spaces in new and modified lines in a filename'''
+    '''Remove trailing white spaces in new and modified lines in a filename
+
+    :param filename: The file to check
+    :param new_file: True if whole file is new; False if it's an existing
+        file that's been modified
+    :param in_place: True if trailing whitespace is to be removed from file in
```

Comment:
I find the wording of the variable doesn't help me understand what's happening. I've seen dry_run used as an expression for this. You could also call it do_not_change_files

Reply:
Will change to dry_run.